uint8 START_CALIB = 1
uint8 STOP_CALIB = 0

uint8 status
---
bool ack
